doc/man3/BN_mod_inverse.pod

```diff
@@ -2,7 +2,8 @@
 
 =head1 NAME
 
-BN_mod_inverse - compute inverse modulo n
+BN_mod_inverse, BN_are_coprime -
+inverse modulo operations on BIGNUMs
```

Comment:
BN_are_coprime is **not** a modulo inverse.
This needs a better title.

Reply:
The way it is implemented it is exactly an BN_mod_inverse call. Apart from putting this in a separate file there is no great place to put this..
I think this is sufficient given that the actual function describes what it does.

BN_gcd() is in  BN_add.pod (which it describes as 'arithmetic operations on BIGNUMS'). Not sure it belongs there..

Reply:
I'd be happier with BN_are_coprime being considered arithmetic.
Better still might be in BN_cmp.pod which is _BIGNUM comparison and test functions_ which it is.

Reply:
BN_cmp is a better place.. thanks for the suggestion.